data_library/processed_data/covid_australia/parse_data.py

```diff
@@ -38,7 +38,8 @@ def write_state_data(state, start_date='2020-03-01', end_date='2020-05-01'):
     """
     # Select data from the given state
     data = pandas.read_csv(
-        os.path.join(os.path.dirname(__file__), 'cases.csv'))
+        os.path.abspath(os.path.join(os.path.dirname(__file__), 
+        '../..', 'raw_data/cases_australia.csv')))
```

Comment:
```suggestion
        os.path.abspath(os.path.join(
            os.path.dirname(__file__),
            os.pardir,
            os.pardir,
            'raw_data',
            'cases_australia.csv')))
```
What you've put is working fine, but this could be slightly more robust.